kickstarts/base.ks.erb

```diff
@@ -1,5 +1,8 @@
 # Kickstart file automatically generated by anaconda.
-# Run the text based installer to see dependency info in screenshots
+
+# By default run the text based installer to see dependency info in screenshots
+<% if ENV['KS_GUI'] == '1' then "" else "text" end %>
+
```

Comment:
@jrafanie there is no impact (positive or negative) on image size. The benefit of doing a text mode install is a slightly faster install (2 mins) and the fact that you see a log of what's happening on the screen.